crates/supervisor/core/src/chain_processor/task.rs

```diff
@@ -348,6 +355,17 @@ where
                     }
                     return Err(StorageError::BlockOutOfOrder.into());
                 }
+
+                Err(StorageError::ReorgRequired) => {
+                    error!(
+                        target: "chain_processor",
+                        chain = self.chain_id,
+                        derived_block = %derived_ref_pair.derived,
+                        "Local derivation conflict detected — rewinding"
+                    );
```

Comment:
this should be a debug log. a reorg isn't a reason to encourage manually stopping the node for debugging, which is the call to action that the error log level should be reserved for.